### ISSUE TITLE
(v0.9.5) Not installable via Chrome Web Store

### ISSUE THREAD

**Issue by motin** [mozilla-extensions/regrets-reporter#2]
Steps to reproduce:
* Attempt to install the extension via Chrome Web Store

Expected results:
* The extension installs successfully

Actual results:
* The following error message appears:
```
An error has occurred
Could not decode image: 'icon-toolbar-inactive.svg'
Reload
Close
```

Notes:
* This issue does not show up when the extension is installed without involving Chrome Web Store.
* Seemingly the same issue as https://github.com/single-spa/single-spa-inspector/issues/15
* Chromium bug: https://bugs.chromium.org/p/chromium/issues/detail?id=29683

**Comment by motin** [comment]
A fix has been committed in https://github.com/mozilla-extensions/regrets-reporter/commit/5c744d5c496f409d190c6e693f0047b1d5ddc22e

**Comment by motin** [comment]
The fix was deployed in v0.9.7 and confirmed to be fixed.
